src/components/WalletModal/index.tsx

```diff
@@ -223,10 +223,28 @@ const ModalContent = (props: any) => {
                 }
                 // likewise for generic
                 else if (option.name === 'Injected' && isMetamask) {
-                    return null
+                  return null
```

Comment:
this is used when both would be rendered! but as discussed, for now, only metamask would work in this case.
but I left it there in case we do want to support both simultaneously 

Coinbase will follow the flow of metamask when only coinbase exists
Comment on TG was: Should the icons and name be added for Coinbase?